gulpfile.js

```diff
@@ -22,3 +25,50 @@ gulp.task('integrate:win', function () {
   });
 
 });
+
```

Comment:
It looks like nearly identical changes to this file were made in 5ea845bd7460d4bb3d2a269cd7a225b384754420 by @jeffreytu and in 8cc5e52412f2ba4a450a55612ca92520dec40a2a by @goldlilys.  I'm not sure where to code review these changes because I'm not sure who the author was.

Copy-pasting between commits is not advisable because it's difficult to infer context (i.e. who made the changes and when they were made).  This is one of those times where a separate pull request for these common changes would have been a good idea.  When that's not possible, using `git cherry-pick` to steal a commit and put it in your branch is better than copy-pasting because it maintains authorship information.